TrafficCapture/captureOffloader/src/main/java/org/opensearch/migrations/trafficcapture/model/OffloaderInput.java

```diff
@@ -0,0 +1,24 @@
+package org.opensearch.migrations.trafficcapture.model;
+
+import com.google.protobuf.CodedOutputStream;
+
+import java.util.concurrent.CompletableFuture;
+
+public class OffloaderInput {
+
+    private CodedOutputStream codedOutputStream;
+    private CaptureOutputStreamMetadata captureOutputStreamMetadata;
```

Comment:
See the comment above about inlining the contents of the metadata into this class.